crypto/src/address.rs

```diff
@@ -8,7 +8,7 @@ use crate::{fmd, ka, keys::Diversifier, Fq};
 
 pub const CURRENT_CHAIN_ID: &str = "penumbra-tn001";
 /// Human-readable prefix in the address.
-pub const CURRENT_NETWORK_ID: &str = "penumbra_tn001_";
+pub const CURRENT_NETWORK_ID: &str = "penumbra_testv0_";
```

Comment:
what about `penumbratv0`? that way it's a bit shorter and there's no underscores. it'll run into the leading `1` but i don't think that's really a problem

Reply:
hmm true, better to have shorter addresses, will update